vignettes/migration.Rmd

```diff
@@ -17,7 +17,7 @@ Plumber underwent a series of breaking changes as a part of the 0.4.0 release. T
 
 There are a number of changes that users should consider when preparing to upgrade to `plumber` 0.4.0.
 
-1. **Plumber no longer accepts external connections by default**. The `host` parameter for the `run()` method now defaults to `127.0.0.1`, meaning that Plumber will only listen for incoming requests from the local machine on which it's running -- not from any other machine on the network. This is done for security reasons so that you don't accidentally expose a Plumber API that you're developing to your entire network. To restore the old behavior in which Plumber listened for connections from any machine on the network, use `$run(host="0.0.0.0")`. Note that if you're deploying to an environment that includes an HTTP proxy (such as the [DigitalOcean](./hosting.html#digitalocean) servers which use nginx), having Plumber listen only on `127.0.0.1` is likely the right default, as your proxy -- not Plumber -- is the one receiving external connections.
+1. **Plumber no longer accepts external connections by default**. The `host` parameter for the `run()` method now defaults to `127.0.0.1`, meaning that Plumber will only listen for incoming requests from the local machine on which it's running -- not from any other machine on the network. This is done for security reasons so that you don't accidentally expose a Plumber API that you're developing to your entire network. To restore the old behavior in which Plumber listened for connections from any machine on the network, use `$run(host="0.0.0.0")`. Note that if you're deploying to an environment that includes an HTTP proxy (such as the DigitalOcean servers which use nginx), having Plumber listen only on `127.0.0.1` is likely the right default, as your proxy -- not Plumber -- is the one receiving external connections.
```

Comment:
```suggestion
1. **Plumber no longer accepts external connections by default**. The `host` parameter for the `run()` method now defaults to `127.0.0.1`, meaning that Plumber will only listen for incoming requests from the local machine on which it's running -- not from any other machine on the network. This is done for security reasons so that you don't accidentally expose a Plumber API that you're developing to your entire network. To restore the old behavior in which Plumber listened for connections from any machine on the network, use `$run(host="0.0.0.0")`. Note that if you're deploying to an environment that includes an HTTP proxy (such as the [DigitalOcean](./hosting.html#digitalocean) servers which use nginx), having Plumber listen only on `127.0.0.1` is likely the right default, as your proxy -- not Plumber -- is the one receiving external connections.```